docs/operator_tutorial.md

```diff
@@ -0,0 +1,209 @@
+
+Continuing with the user profile example, the `postalCode` property is currently a string (text).  Since it would be more helpful for users to search for profiles in the southeastern United States (300000-39999), you'll convert the `postalCode` to an integer using the **convert** operator.  **Convert** is placed between flatten and insert.
+
+```javascript
+{
+  id: 'fs-identify-ceddl-user-zip',
+  description: 'send CEDDL user with numeric zip to FS.identify',
+  source: 'digitalData.user.profile[0]',
+  operators: [{ name: "flatten" }, { name: "convert", properties: "postalCode", type: "int" }, { name: "insert", select: "profileID" }],
+  destination: 'FS.identify',
+}
+```
+
+This produces a result where only the specified `properties` have been converted to the desired `type`.
+
+```javascript
+{
+ profileID: 'pr-12333211',
+ userName: 'JohnyAppleseed',
+ line1: '123 Easy St.',
+ line2: '',
+ city: 'Athens',
+ stateProvince: 'GA',
+ postalCode: 30606,
+ country: 'USA'
+}
+```
+
+## Filtering and Choosing Data
+
+Let's do one last set of transformations to reduce the amount of information collected based on a few business requirements.
+
+- Identifying users should only be done on a checkout page since that requires the user to be logged in.
+- The business doesn't need to record address information since the `profileID` can be used to look up customer records.
+- The business would also prefer to identify users only if they are located in the United States, which is denoted by the `country` property.
+
+To accomplish the above, you'll leverage the **query** operator.  **Query** can be used to pick or omit properties from an object.  The decision to use pick versus omit is that former allows *only* the properties you specify while the latter allows *any* properties but *removes* the properties specified.  If there's a chance a developer could add an unwanted property in the future, you'd leverage pick over omit.  The below rule uses omit.
+
+Filtering is added also using the **query** operator and selector syntax.  While not shown, there are more comparisons that equality (e.g. greater than, not equal, etc).
+
+```javascript
+{
+       id: 'fs-identify-ceddl-user-no-address',
+       description: 'send CEDDL user with numeric zip to FS.identify',
+       source: 'digitalData.user.profile[0]',
+       operators: [
+         { name: "flatten" },
+         { name: "query", select: "$[?(country=USA)]" },
+         { name: "query", select: "$[!(line1,line2)]" },
+         { name: "convert", properties: "postalCode", type: "int" },
+         { name: "insert", select: "profileID" }],
+       destination: 'FS.identify'
+     }
+```
+
+The result of this rule is a call to `FS.identify` with only the desired properties under the desired circumstances.
+
+## Next Steps
+
+A full list of the available operators and how to configure them is found in the Operator Reference.  Many examples also exist in the rules directory of this repository.
```

Comment:
It might be handy to include the operator table from the Readme here as well, with links to each individual doc.